packages/preview/hsrm-bachelor-thesis/0.6.0/typst.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "hsrm-bachelor-thesis"
```

Comment:
Regarding the name of this package and of the other templates, they sound like they are official. You should chose other names that are less canonical.